Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -76,14 +76,24 @@ void Wiimote::Reset()
   // EEPROM
   m_eeprom = {};
 
-  // IR calibration and maybe more unknown purposes:
-  // The meaning of this data needs more investigation.
-  // Last byte is a checksum.
-  std::array<u8, 11> ir_calibration = {
-      0xA1, 0xAA, 0x8B, 0x99, 0xAE, 0x9E, 0x78, 0x30, 0xA7, 0x74, 0x00,
-  };
-  // Purposely not updating checksum because using this data results in a weird IR offset..
-  // UpdateCalibrationDataChecksum(ir_calibration, 1);
+  // IR calibration:
+  std::array<u8, 11> ir_calibration = {// Point 1
```

Comment:
You may want to consider formatting this like:

```cpp
std::array<u8, 11> ir_calibration{
      // Point 1
      IR_LOW_X & 0xFF,
      IR_LOW_Y & 0xFF,

      // Mix
      ((IR_LOW_Y & 0x300) >> 2) | ((IR_LOW_X & 0x300) >> 4) | ((IR_LOW_Y & 0x300) >> 6) |
          ((IR_HIGH_X & 0x300) >> 8),

      // Point 2
      IR_HIGH_X & 0xFF,
      IR_LOW_Y & 0xFF,

      // Point 3
      IR_HIGH_X & 0xFF,
      IR_HIGH_Y & 0xFF,

      // Mix
      ((IR_HIGH_Y & 0x300) >> 2) | ((IR_HIGH_X & 0x300) >> 4) | ((IR_HIGH_Y & 0x300) >> 6) |
          ((IR_LOW_X & 0x300) >> 8),

      // Point 4
      IR_LOW_X & 0xFF,
      IR_HIGH_Y & 0xFF,

      // Checksum
      0x00,
  };
```

So that it doesn't look wonky.

Reply:
Agreed that the current one is wonky.  This is the formatting that the auto-formatter gave, and I'm not sure how much leeway it gives... but I'll try that and see if it accepts it.

Reply:
Oh, interesting!  At least with visual studio, it produces the current jank one without the trailing comma, but adding the comma gives what you have which is much more readable.